### ISSUE TITLE
Docs improvements

### ISSUE THREAD

**Issue by davestewart** [davestewart/vuex-pathify#64]
Follow up on comments left here:

- https://www.reddit.com/r/vuejs/comments/bydml3/if_you_like_vuex_but_hate_working_with_vuex/eqjhyf2/

**Comment by hunterae** [comment]
Hey @davestewart ,
One small clarification: whenever I mention "landing page" above, I'm referring to the documentation landing page, whereas it appears I refer to the Github Readme page as either "Github landing page" or "Github page" or "Github" readme. Sorry if this causes any confusion.

Also, you mentioned in reply on the Reddit thread, there are definitely discrepancies between the Github Readme and the documentation site root page. I came across one possible solution for dealing with that problem when I was figuring out how to assemble my own documentation for my [VueTableFor](https://hunterae.github.io/vue-table-for/#/) plugin (I know that documentation itself needs a ton of work; also, I just realized I never pushed that change so you'll see a commit for it today). Essentially (and assuming you're generating your documentation using Vue), you can have your main documentation component import your README.md file using vue-markdown-loader and then render it. See l[ink 1](https://github.com/hunterae/vue-table-for/blob/d0d4a6a1b2c306b0dab84d19d4989ff962392808/examples/App.vue#L18) and [link 2](https://github.com/hunterae/vue-table-for/blob/d0d4a6a1b2c306b0dab84d19d4989ff962392808/examples/App.vue#L34-L43) (Note: I had to import some specific css styles to display the Markdown nicely). 

For purposes of credit, I saw that approach used on the [vue-fullscreen plugin documentation](https://github.com/mirari/vue-fullscreen).

Regarding the "@" syntax in paths, as you mentioned, a switch to only using "." would probably be the best case scenario. For the short-term, I'm actually fine with the "@" syntax (and was fine using it once I understood what it was doing). My brain just found it hard to parse the visual example in the documentation of "bar/baz@a.b.c". I remember looking at that several times previously asking myself, ok, what's the module, what's the state field, so then what's "a" and "b" and "c". It took me an embarrassingly long time to internalize the information. I'm sure you're already thinking along these lines, but perhaps V2, as well as the documentation, can support both syntaxes. That way, you cater towards the people who view it as an object lookup chain vs the people who want to be explicit in their sub-object lookup. I think both approaches could be easily justified.

**Comment by pacific202** [comment]
I'm new to pathify, but am excited about how it's going to greatly simplify my current project which is very complex and will rely on vuex extensively.  The vuex docs state that you will know when you need state management, and boy do I ever need it on this one.

I've read most of the feedback above, and for the most part agree.  I won't address the nuances of that as I don't want to impede progress in any way.

My struggle with pathify is moving from a very simple "Hello World" demo up to something that is far more complex and doesn't seem to have a real-world style of scenario.

I'm also happy to help, perhaps by converting the vuex examples (shopping cart et. al.) to pathify.
